Description: Disable an unreliable test
 This test is currently unreliable due to slight differences in
 floating-point representation (https://github.com/ekmett/lens/issues/310),
 and fails on hurd-i386 and kfreebsd-i386 in a way that does not indicate a
 problem with the **~ operator itself.  This patch may be dropped once the
 test is fixed.
Author: Colin Watson <cjwatson@debian.org>
Forwarded: no
Last-Update: 2013-06-17

Index: b/src/Control/Lens/Setter.hs
===================================================================
--- a/src/Control/Lens/Setter.hs
+++ b/src/Control/Lens/Setter.hs
@@ -669,9 +669,6 @@
 -- >>> (a,b) & both **~ c
 -- (a**c,b**c)
 --
--- >>> _2 **~ pi $ (1,3)
--- (1,31.54428070019754)
---
 -- @
 -- ('**~') :: 'Floating' a => 'Setter'' s a    -> a -> s -> s
 -- ('**~') :: 'Floating' a => 'Iso'' s a       -> a -> s -> s
